src/examples/unique-taxpayer-reference/corporation-tax/index.njk

```diff
@@ -7,9 +7,9 @@ layout: layout-example.njk
 
 <h1 class="govuk-heading-l">What is your Corporation Tax Unique Taxpayer Reference (UTR)?</h1>
 
-<p class="govuk-body">Your UTR can be 10 or 13 digits long. You can <a class="govuk-link" rel="noreferrer noopener" target="_blank" href="https://www.gov.uk/guidance/sign-in-to-your-hmrc-business-tax-account">find it in your Business Tax Account (opens in new tab)</a>, on tax returns and other documents from HMRC. It might be called ‘reference’, ‘UTR’ or ‘official use’.</p>
+<p class="govuk-body">Your UTR can be 10 or 13 digits long. It will be on tax returns and other letters about Corporation Tax. It might be called ‘reference’, ‘UTR’ or ‘official use’.</p>
 
-<p class="govuk-body"><a class="govuk-link" rel="noreferrer noopener" target="_blank" href="https://www.gov.uk/find-lost-utr-number">Find a lost UTR (opens in new tab)</a>.</p>
+<p class="govuk-body"><a class="govuk-link" rel="noreferrer noopener" target="_blank" href="https://www.tax.service.gov.uk/ask-for-copy-of-your-corporation-tax-utr?_ga=2.1519665.1414616363.1698396656-2042390522.1690192300">Ask for a copy of your Corporation Tax UTR (opens in new tab)</a></p>
```

Comment:
Can you get rid of the query string (everything from the '?' onwards) as that's a Google Analytics event that relates to you machine.